src/NuGet.Core/NuGet.CommandLine.XPlat/ReportRenderers/JsonRenderer/JsonOutputContent.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.CommandLine.XPlat.ReportRenderers.JsonRenderer
+{
+    internal class JsonOutputContent
```

Comment:
Why don't just create an instance of this object and serialize it to json using a well-known json library (System.Text.Json or Newtonsoft.Json) ?

Reply:
Sure, we can tweak serializer (with JsonConverter) later but I'm more worried about `IRenderer.cs` and `ConsoleWriter.cs`, `ListPackageJsonRenderer.cs` etc which is hard to change later. Please take a look at current design.